scripts/release_info.py

```diff
@@ -21,13 +21,17 @@
 
 import argparse
```

Comment:
Try to find a way to add unit tests for as many functions in this file as possible.

Reply:
Done.